app/views/forms/view/directives/view-reference-records.directive.js

```diff
@@ -6,6 +6,7 @@ import '~/services/main';
 import '~/views/forms/view/bch/icons';
 import { iconFields } from '~/views/forms/view/bch/icons';
 import viewReferenceRecordsT from '~/app-text/views/forms/view/directives/view-reference-records.json';
+import { getFieldName } from '../../../../services/getFieldName';
```

Comment:
you can use home `~` instead of ../../..
```suggestion
import { getFieldName } from '~/services/getFieldName';
```